### PR TITLE
minio: updated to 2019-09-26T19-42-35Z and go to 1.13 (security update)

### PR DESCRIPTION
**Pull request by fgma**
_Motivation:_  Security release for minio
_Linked issues:_  https://github.com/minio/minio/releases/tag/RELEASE.2019-09-26T19-42-35Z

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

Security fix for minio. To build current minio release go needs to be updated to 1.13.

### CODE REVIEW

File: spk/minio/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 SPK_NAME = minio
-SPK_VERS = 2019-08-21T19-40-07Z
+SPK_VERS = 2019-09-26T19-42-35Z
 SPK_REV = 1
```
Comment (hgy59) [inline spk/minio/Makefile]: please regard:
SPK_REV = Package Revision number. 
Starts at 1, goes up 1 for every subsequent release

Comment (fgma) [reply spk/minio/Makefile]: Thanks, I've missed that. Fixed in https://github.com/SynoCommunity/spksrc/pull/3779/commits/a00d251904d7995a4aedb0ea35102ab2ba5dab18

**Comment by hgy59** [comment]
@fgma do we need the update of go to resolve the security issue?

if NO:
Please remove the update of go for this PR and create a separate PR for the update of go

If YES:
We have to update and validate the build of all packages depending on go (dnscrypt-proxy, syncthing, fritzctl/synocli-file)

Sorry, on the first view I did not recognize the update of go here...

**Comment by fgma** [comment]
@hgy59 Minio fails to compile with the old go version. They've switched to 1.13 in https://github.com/minio/minio/tree/RELEASE.2019-09-11T19-53-16Z.

I tried to compile syncthing spk with the updated go version which worked for me. But I haven't tested it because I don't use it.

What kind of setup are you using for development purpose? I only have access to a single DSM system which I'd like to keep stable. VirtualDSM is also not an option I can use for testing.

**Comment by fgma** [comment]
@hgy59 I've compiled and installed minio, dnscrypt-proxy, fritzctl and syncthing succesfully.

**Comment by fgma** [comment]
@hgy59 I've updated go and minio the latest release and incremented all dependent packages SPK_REV's. Let me know if there is anything left to do.

File: spk/synocli-net/Makefile [hunk 2]
```diff
@@ -17,7 +17,7 @@ MAINTAINER = ymartin59
 DESCRIPTION = "SynoCli Network Tools provides a set of small command-line utilities: screen, tmux, mosh, socat, nmap, sshfs, fritzctl. Credits to Sebastian Schmidt \(publicarray\) for icons"
 DISPLAY_NAME = SynoCli Network Tools
 STARTABLE = no
-CHANGELOG = "3. Build sshfs 2.10 with fuse updated to v2.9.9"
+CHANGELOG = "built with go 1.13.1"
```
Comment (hgy59) [inline spk/synocli-net/Makefile]: only fritzctl is built with go

Comment (fgma) [reply spk/synocli-net/Makefile]: Changed in https://github.com/SynoCommunity/spksrc/pull/3779/commits/c0cd05c187bb78a5088bd0ce9895ae238bd78e07

File: spk/synocli-net/Makefile [hunk 3]
```diff
@@ -17,7 +17,7 @@ MAINTAINER = ymartin59
...
 
 HOMEPAGE   = https://github.com/SynoCommunity/spksrc/wiki/FAQ-SynoCliNet
 LICENSE    = Mixed
```
Comment (hgy59) [inline spk/synocli-net/Makefile]: If you are updating this file:
I'd rather set LICENSE = Each tool is licensed unter its respective license.
_There does not exist a "mixed" license._

Comment (fgma) [reply spk/synocli-net/Makefile]: Changed in https://github.com/SynoCommunity/spksrc/pull/3779/commits/2e581e055375d7545188c1ee72151e71ac3570d9

**Comment by hgy59** [comment]
Just to mention: 
_This issue is CVE-2019-16276 and is fixed in Go 1.13.1 and Go 1.12.10._

With Go 1.13 the default SSL Version is now TLS 1.3 and I hope there are no side effects in the dependent packages (dnscrypt-proxy, syncthing).

**Comment by hgy59** [comment]
And there is already #3771 for updating syncthing

**Comment by hgy59** [comment]
And maybe we integrate ser2net of #3733 into synocli-net

**Comment by fgma** [comment]
I've removed all changes to syncthing an synocli-net.

**Comment by hgy59** [comment]
@ymartin59 shall I merge this - or is it enough to label `status/ready to merge` and you will merge and push new versions at your schedule?

**Comment by fgma** [comment]
@hgy59 Just merge it.

**Comment by hgy59** [comment]
@fgma merging does not create new versions in the package repository (as you might expect) and sometimes it is better to keep a certain order in merging PRs to avoid merge conflicts. And this would be my first merge (except one that I did accidentially) therefore I want to have the OK of @ymartin59.

As far as I know only @ymartin59 and @Diaoul are given access to push packages to the package repo.

**Comment by fgma** [comment]
@hgy59 No problem, we just wait until someone else approves it. I thought all merges to master will automatically produce updated builds.

I'll not change anything in this PR anymore expect for potential new minio releases until it gets merged.

**Pull request merged**